### ISSUE TITLE
A11y_Manage.IIS.Net_Setup Progress Popup: Color Contrast ratio fails for the heading "Microsoft IIS Administration 2.2.0 Setup" which is 2.4:1

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/IIS.WebManager#209]
**Environment Details:**
**Browser:** Microsoft Edge 42.17134.1.0
**WCAGID Violated**:1.4.3

**Repro Steps:-** 
1.Hit the URL and login with appropriate credentials and application landing page should open
2. Tab till "Download Microsoft IIS Administration" button and press enter and run "IISAdministrationSetup.exe" file
3. Check the check box in the “Microsoft IIS Installation Popup” and tab till "Install" button and press enter.
4.Use Color Contrast Analyzer and check color contrast ratio for all content and controls

**Actual Result:-**
Color Contrast ratio fails for the heading "Microsoft IIS Administration 2.2.0 Setup" which is 2.4:1

**Expected Result:-**
Color Contrast ratio fails for the heading "Microsoft IIS Administration 2.2.0 Setup" which is 2.4:1 should be greater than or equal to 4.5:1

**User Impact:-**
Users with visual disabilities finds it difficult in viewing controls if they doesn’t meet color contrast ratio standards

**Attached Screenshot for your reference:**
![bug 209_color contrast ratio of heading](https://user-images.githubusercontent.com/39204909/46283189-4f875880-c591-11e8-96c0-378c06220ac9.png)

**Comment by jhkimnew** [comment]
Duplicated https://github.com/Microsoft/IIS.Administration/issues/221

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ColorContrast; #MAS1.4.3; #C+E-H; #.NETWebsites; #Edge

**Comment by prabhudevu** [comment]
**GitHubTags:**#A11ySev2;

**Comment by BhanuPrakashGit** [comment]
GithubTag:#Closed;
manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](https://nam06.safelinks.protection.outlook.com/?url=http%3A%2F%2Fwww.iis.net%2F&data=05%7C01%7Cv-bhaprakash%40microsoft.com%7C7a737faa4af14c4ad31a08db634ab100%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638212945750407456%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Q7lvppcGrDLIRxk6EaLiumHk2i6E5S784T%2BniobLEu4%3D&reserved=0) home page will be removed.
